### ISSUE TITLE
[BUG] Close Button Not Working Properly on Installation Options Dialog

### ISSUE THREAD

**Issue by tEctutoRIt** [marticliment/UniGetUI#2403]
### Please confirm these before moving forward

- [X] I have searched for my issue and have not found a work-in-progress/duplicate/resolved issue.
- [X] I have tested that this issue has not been fixed in the latest [(beta or stable) release](https://github.com/marticliment/WingetUI/releases/).
- [X] I have checked the [FAQ](https://github.com/marticliment/WingetUI#frequently-asked-questions) section for solutions.
- [X] This issue is about a bug (if it is not, please use the [correct template](https://github.com/marticliment/WingetUI/issues/new/choose)).

### UniGetUI Version

3.1.0

### Windows version, edition and architecture

Windows 11 Pro 22631.3810

### Describe your issue

Clicking the X (close icon) on red box does not close the dialog box in the Installation Options dialog box, only clicking a small portion at the lower side of the X icon closes it. But works perfectly in fullscreen.

### Steps to reproduce the issue

Open UniGetUI in normal state (not fullscreen)
Go to the Software Updates tab.
Open the Installation Options dialog box via icon.
Click the red X (close icon) at the top right.

### UniGetUI Log

```text
[03/07/2024 7:06:45 am] 
                           __  __      _ ______     __  __  ______
                          / / / /___  (_) ____/__  / /_/ / / /  _/
                         / / / / __ \/ / / __/ _ \/ __/ / / // /  
                        / /_/ / / / / / /_/ /  __/ /_/ /_/ // /   
                        \____/_/ /_/_/\____/\___/\__/\____/___/   
                            Welcome to UniGetUI Version 3.1.0
[03/07/2024 7:06:45 am]
[03/07/2024 7:06:45 am] Version Code:  3.1
[03/07/2024 7:06:46 am] An error occurred while migrating chocolatey
[03/07/2024 7:06:46 am] System.ArgumentException: The path is empty. (Parameter 'relativeTo')
                           at System.IO.Path.GetRelativePath(String relativeTo, String path, StringComparison comparisonType)
                           at UniGetUI.PackageEngine.Managers.ChocolateyManager.Chocolatey.LoadManager() in C:\SomePrograms\WingetUI-Store\src\UniGetUI.PackageEngine.Managers.Chocolatey\Chocolatey.cs:line 278
[03/07/2024 7:06:46 am] Lang files were updated successfully from GitHub
[03/07/2024 7:06:46 am] Downloaded new icons and screenshots successfully!
[03/07/2024 7:06:47 am] Command scoop was not found on the system
[03/07/2024 7:07:03 am] Backup saved to C:\Users\73djj\Documents\UniGetUI\GUY_FAWKES installed packages 2024-07-03 07-07-03.json
```

### Package Managers Logs

```text
Manager Winget with version:
Naive WinGet CLI Version: v1.8.1791
Microsoft.WinGet.Client PSModule version: 
Using Native WinGet helper (COM Api)

——————————————————————————————————————————


Logged native task on manager Winget. Task type is ListSources
Process start time: 03/07/2024 7:19:07 am
Process end time:   03/07/2024 7:19:07 am

-- Task information
 ...

The task reported success

——————————————————————————————————————————

Logged subprocess-based task on manager Winget. Task type is RefreshIndexes
Subprocess executable: "C:\Users\73djj\AppData\Local\Microsoft\WindowsApps\winget.exe"
Command-line arguments: " source update --disable-interactivity"
Process start time: 03/07/2024 7:19:11 am
Process end time:   03/07/2024 7:19:14 am

-- Process STDOUT
 ...

Return code: SUCCESS (0)

——————————————————————————————————————————

Logged subprocess-based task on manager Winget. Task type is ListPackages
Subprocess executable: "C:\WINDOWS\system32\windowspowershell\v1.0\powershell.exe"
Command-line arguments: "-ExecutionPolicy Bypass -NoLogo -NoProfile"
Process start time: 03/07/2024 7:19:11 am
Process end time:   UNFINISHED

-- Process STDIN
 ...

-- Process STDOUT
 ...

Return code: Process has not finished yet

——————————————————————————————————————————

Logged native task on manager Winget. Task type is ListSources
Process start time: 03/07/2024 7:19:12 am
Process end time:   03/07/2024 7:19:12 am

-- Task information
 ...

The task reported success

——————————————————————————————————————————

Logged subprocess-based task on manager Winget. Task type is ListUpdates
Subprocess executable: "C:\WINDOWS\system32\windowspowershell\v1.0\powershell.exe"
Command-line arguments: "-ExecutionPolicy Bypass -NoLogo -NoProfile"
Process start time: 03/07/2024 7:19:14 am
Process end time:   UNFINISHED

-- Process STDIN
 ...

-- Process STDOUT
 ...

Return code: Process has not finished yet

——————————————————————————————————————————
```

### Relevant information

This issue does not affect all dialog boxes.
The problem is specific to Installation Options dialog box.

### Screenshots and videos

![WingetUI_4Ttc7E4QPG](https://github.com/marticliment/UniGetUI/assets/77356742/ae01a77d-2c74-4b80-bdb7-a14abb72e3d4)

**Comment by marticliment** [comment]
What happens if you resize the UniGetUI window so it is a little bit taller?

**Comment by marticliment** [comment]
I suspect it is interfering with the title bar

**Comment by marticliment** [comment]
This issue was fixed on AppSdk 1.6.

Latest UniGetUI beta uses AppSdk 1.6
